### PR TITLE
Add documentation for customize-image-cropper.js

### LINKED ISSUE #69: Document controllers/customize-image-cropper.js

### PR DESCRIPTION
**Pull request by manuelaugustin**
Fixes #69 
Please give props to @BramHeijmink, @manuelaugustin and @ireneyoast

### CODE REVIEW

File: src/wp-includes/js/media/controllers/customize-image-cropper.js [hunk 1]
```diff
@@ -4,14 +4,20 @@ var Controller = wp.media.controller,
...
+ * @since 4.3
+ *
+ * @memberOf wp.media.controller
  *
  * @class
  * @augments wp.media.controller.Cropper
  * @augments wp.media.controller.State
  * @augments Backbone.Model
+ *
+ * @returns ajax post Call to Ajax to crop the image according to the crop details.
```
Comment (andizer) [inline src/wp-includes/js/media/controllers/customize-image-cropper.js]: I guess this should a `jQuery.promise`.  See : https://github.com/Yoast/wordpress-develop-mirror/pull/82#discussion_r147137062

File: src/wp-includes/js/media/controllers/customize-image-cropper.js [hunk 2]
```diff
@@ -4,14 +4,20 @@ var Controller = wp.media.controller,
 /**
  * wp.media.controller.CustomizeImageCropper
  *
- * @memberOf wp.media.controller
+ * @summary Prepares the image to get cropped.
```
Comment (boblinthorst) [inline src/wp-includes/js/media/controllers/customize-image-cropper.js]: This summary is not a summary of the longer description. Also, when the 'longer' description is only one line, you don't need to add an @summary.

File: src/wp-includes/js/media/controllers/customize-image-cropper.js [hunk 3]
```diff
@@ -4,14 +4,20 @@ var Controller = wp.media.controller,
...
+ *
+ * Checks whether the image needs to be cropped according to a ratio.
  *
- * A state for cropping an image.
+ * @since 4.3
```
Comment (boblinthorst) [inline src/wp-includes/js/media/controllers/customize-image-cropper.js]: Always use an x.x.x format (4.3.0)

File: src/wp-includes/js/media/controllers/customize-image-cropper.js (hunk 1 above)
Comment (boblinthorst) [inline src/wp-includes/js/media/controllers/customize-image-cropper.js]: Please add type and remove 'ajax post'

File: src/wp-includes/js/media/controllers/customize-image-cropper.js [hunk 4]
```diff
@@ -4,14 +4,20 @@ var Controller = wp.media.controller,
...
  *
  * @class
  * @augments wp.media.controller.Cropper
  * @augments wp.media.controller.State
  * @augments Backbone.Model
+ *
+ * @returns ajax post Call to Ajax to crop the image according to the crop details.
  */
 CustomizeImageCropper = Controller.Cropper.extend(/** @lends wp.media.controller.CustomizeImageCropper.prototype */{
 	doCrop: function( attachment ) {
```
Comment (boblinthorst) [inline src/wp-includes/js/media/controllers/customize-image-cropper.js]: Please document this function as well

**Comment by IreneStr** [comment]
CR 👍

**Comment by atimmer** [comment]
Committed in https://core.trac.wordpress.org/changeset/42435.

**Pull request closed without merging**